### PR TITLE
Fixes to equivalencies (sorting+few changes)

### PR DESCRIPTION
**Pull request by Jalopyy**
Just numerically sorting a few I forgot and added a thing or two I forgot (like adding Holy Ward to ImmunetoMagicCC)

### CODE REVIEW

**Comment by Jalopyy** [comment]
I also added a spellID or two that won't exist until 8.1, hopefully that won't cause any issues by referencing them before the patch drops?

**Comment by Jalopyy** [comment]
What is the 'MiscHelpfulBuffs' supposed to be for? It has freedom effects in there, but then it also has a few random speed boosts?

**Comment by ascott18** [comment]
I'd actually be OK with removing that one. It is so confused in its purpose that I really doubt many people are using it. I'll take care of doing that since I'll need to write a migration for it.

- [ ] Cyb: Remove MiscHelpfulBuffs, write migration.

Its very, very old. Added in 4a729530d75dc2bd48d3f119bc691f03ca048536 and probably never touched since.

**Comment by ascott18** [comment]
Non-existent IDs are fine. They get ignored if the spell don't exist.

**Comment by Jalopyy** [comment]
I'll remove the miscbuffs in a bit. Might add a section for aura mastery and freedom effects.

**Comment by ascott18** [comment]
Just to be clear, I don't want you to remove it. I'll take care of it because there's other work to be done beyond just deleting it.

**Comment by Jalopyy** [comment]
Oh, ok. I'll leave it be then.

**Comment by Jalopyy** [comment]
Will probably still make an aura mastery list, but probably won't make a freedom one. There aren't too many and a few are only freedom from snares instead of roots. Full immunities, too. Some things like BOP provide immunity to physical kicks (pummel, skull bash, etc) but not magical ones (mind freeze, counterspell, spell lock).

**Comment by Jalopyy** [comment]
Added the ImmuneToInterrupts. I could add ones for ImmuneToMagic/PhysInterrupts. Your call, not sure if that is getting too specific. Could be niche useful for people who want to make an aura for when they can't kick which is dependent on the class they're on.

**Comment by Jalopyy** [comment]
Similar to MiscHelpfulBuffs - what are PvPSpells for?

**Comment by Jalopyy** [comment]
Casting Circle, which when talented makes it so Unending Resolve no longer gives you kick immunity, does not change the spellID of UR, unfortunately. This will create false positives unless you have some hacky way of creating some internal if/then check that seeing Casting Circle casts removes Unending Resolve from the group for the duration of that arena match or whatnot.

**Comment by Jalopyy** [comment]
Removed Evocation/Rebirth/Roots from PVPSpells. Evocation isn't a big deal (and arcane is very niche, so might readd it later), rebirth can only be cast in world PVP, and entangling roots aren't really a "high-value" CC to interrupt, at least in my opinion.

What do you want your definition of a high-value interrupt to be? Strict-ish as to what my definition might be? Or lax to include basically any casted CC (poly/rep/hex/etc), nuke (chaos bolt/greater pyro), or hard hitting ability (ray of frost, glacial spike)?

File: Components/Core/Spells/Equivalencies.lua [hunk 1]
```diff
@@ -456,6 +503,17 @@ TMW.BE = {
 			 256740, -- Drums of the Maelstrom
 			 264667, -- Primal Rage
 		},
+		ImmuneToInterrupts = {
+			    642, -- Divine Shield
+			 186265, -- Aspect of the Turtle
+			 196773, -- Inner Focus
+			 209584, -- Zen Focus Tea
+			 210294, -- Divine Favor
+			 221705, -- Casting Circle
```
[the remark this answers is not in the document]
Comment (ascott18) [reply Components/Core/Spells/Equivalencies.lua]: Oh, got it. I'm not familiar with the spell and didn't bother to actually look it up. Sorry!

**Comment by ascott18** [comment]
TODO for me:
- [ ] Remove PvPSpells
- [ ] Remove MiscHelpfulSpells

**Comment by Jalopyy** [comment]
If you wanted to retrofit PvPSpells, I could turn it into high priority buffs to watch on yourself or others that don't fit into a CC category.

IE vendetta, duel, sharpen blade, ursol's vortex, etc.

**Comment by ascott18** [comment]
PvPSpells is a spell cast equivalency, not a buff/debuff equivalency.

**Comment by Jalopyy** [comment]
Can't I just rename it and be on my way? Unless I understand less about LUA than I think, it doesn't seem to be formatted any differently than any of the other groups which are buff/debuff equivalencies.

**Comment by Jalopyy** [comment]
Pushed some last minute changes - if this isn't alright since the changes were already approved, I can make a new PR.

**Pull request merged**